src/main/java/com/iridium/iridiumskyblock/database/User.java

```diff
@@ -122,7 +122,7 @@ public LocalDateTime getJoinTime() {
      *
      * @return The player object if one was found, null otherwise
      */
-    public Player toPlayer() {
+    public Player getPlayer() {
```

Comment:
I don't really see a reason for changing this? It will just break the API when it's completely unnecessary.

Reply:
toPlayer is weird, getPlayer is better and if it does break api's then like um yeah